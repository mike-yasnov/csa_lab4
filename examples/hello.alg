// Простая программа Hello World

function main() {
    print("Hello, World!");
}

main(); 