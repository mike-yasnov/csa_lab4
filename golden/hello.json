{
  "return_code": 0,
  "stdout": "Загрузка программы: /tmp/tempfile/program.bin\nЗагружено 7 инструкций\nЗагрузка данных: /tmp/tempfile/program_data.bin\nЗагружено 14 байт данных\nЗапуск выполнения (максимум 1000000 тактов)...\n\n=== РЕЗУЛЬТАТЫ ВЫПОЛНЕНИЯ ===\nСостояние: halted\nВыполнено инструкций: 3\nЗатрачено тактов: 8\nФинальный PC: 2\n\nВЫВОД ПРОГРАММЫ:\nHello, World!\n\nПрограмма завершена успешно.\n",
  "stderr": ""
}